<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manufacturer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Type of plotter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Format type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speed print</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Max widht print</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Price</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuReference">
    <property name="title">
     <string>&amp;Reference</string>
    </property>
    <addaction name="actionAbout_programm"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>&amp;Language</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCharts">
    <property name="title">
     <string>&amp;Charts</string>
    </property>
    <addaction name="actionSpeed_of_Print"/>
    <addaction name="actionPrice"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReference"/>
   <addaction name="menuCharts"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_programm"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-catalog-64.png</normaloff>:/icons/icons8-catalog-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-save-52.png</normaloff>:/icons/icons8-save-52.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-save-as-52.png</normaloff>:/icons/icons8-save-as-52.png</iconset>
   </property>
   <property name="text">
    <string>Sa&amp;ve as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-exit-64.png</normaloff>:/icons/icons8-exit-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-macos-close-60.png</normaloff>:/icons/icons8-macos-close-60.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_programm">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About programm</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-add-64.png</normaloff>:/icons/icons8-add-64.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/icons/icons8-delete-bin-64.png</normaloff>:/icons/icons8-delete-bin-64.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSpeed_of_Print">
   <property name="text">
    <string>&amp;Speed of Print</string>
   </property>
  </action>
  <action name="actionPrice">
   <property name="text">
    <string>&amp;Price</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
